source/index.html.md.erb

```diff
@@ -16,7 +16,8 @@ You can read [non-technical information](https://www.payments.service.gov.uk/usi
 
 - creating an account and selecting a payment service provider (PSP)
 - managing your account with Stripe, GOV.UK Pay's PSP
-- setting up payment links
+- setting up Direct Debit
```

Comment:
we aren't really supporting DD at the moment so I don't think we want to highlight this. We've got the info up to generate interest, but don't think we need to include it in this overview.